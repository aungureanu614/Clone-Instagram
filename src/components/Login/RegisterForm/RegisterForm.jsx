import React from 'react'

const RegisterForm = () => <div>Hi from register form</div>

export default RegisterForm
