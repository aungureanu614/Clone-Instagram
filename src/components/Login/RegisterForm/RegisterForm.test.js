import React from 'react'
import renderer from 'react-test-renderer'
import RegisterForm from './RegisterForm'

test('RegisterContainer renders', () => {
  const component = renderer.create(<RegisterForm />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
