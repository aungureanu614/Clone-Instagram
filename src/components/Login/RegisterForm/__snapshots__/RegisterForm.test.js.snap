// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterContainer renders 1`] = `
<div>
  Hi from register form
</div>
`;
